crates/cxx-qt-gen/src/generator/cpp/mod.rs

```diff
@@ -78,6 +78,38 @@ impl GeneratedCppBlocks {
         })
     }
 }
+/// Returns a vector of the names and types ([CppNamedType] of the parameters of this method, used in cpp generation step
```

Comment:
should be a new line between functions, so add a new line above this comment

Reply:
Done